bin/fixedcase/README.md

```diff
@@ -1,27 +1,49 @@
 # Scripts for marking fixed-case words in BibTeX entries
 
-1. Generate `truelist` by running `train.py import/*.xml`.
-2. For each XML file, run `protect.py <infile> <outfile>`. This marks
-   every fixed-case title word with the tag `<fixed-case>`.
+1. Generate `truelist-auto` by running `train.py import/*.xml`.
+2. Generate `truelist-phrasal-auto` by running `train_phrasal.py import/*.xml`.
+3. Create `truelist` by curating the two lists above.
+4. For each XML file, run `protect.py <infile> <outfile>`. This marks
+   every fixed-case uppercase sequences in title words with the tag `<fixed-case>`.
+   Any existing `<fixed-case>` tags are retained.
 
-Fixed-caseness is determined by these rules:
+Fixed-caseness is determined by this decision list:
 
-- If a word appears in an English lexicon as all-lowercase, it is not
-  fixed-case.
-- If a word appears in `truelist`, it is fixed-case.
-- Any word with a capital letter in a non-initial position (e.g.,
-  `TextTiling`) is fixed-case.
-- If a title is in all caps, the above rule does not apply.
-- If a title starts with a single word set off by a colon or dash, the
-  first word is fixed-case.
+1. If a full title has a case-insensitive match in `special-case-titles`, only
```

Comment:
I've always imagined the caser as being used once and then hand corrected. If the caser has a list like this, that means it's meant to be safe to run twice, right? does that need to be documented anywhere?

Reply:
You mean someone might want to run it, then add entries to special-case-titles, then run it again?

Reply:
I'll move this to the main comment thread...